Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -2723,25 +2724,35 @@ void MolDraw2D::extractSGroupData(const ROMol &mol) {
         Point2D origLoc{xp, yp};
 
         if (fieldDisp[25] == 'R') {
-          origLoc += mol.getConformer().getAtomPos(atomIdx);
+          if (atomIdx < 0) {
+            // we will warn about this below
+            text = "";
+          } else if (fabs(xp) > 1e-3 || fabs(yp) > 1e-3) {
+            origLoc += mol.getConformer().getAtomPos(atomIdx);
```

Comment:
Should we offset this by `xp`, `yp` ?

Reply:
That's already done since `origLoc` was initialized to `(xp,yp)`

Reply:
Ah, right, I missed that.